internal/impl/deploy.go

```diff
@@ -86,6 +87,14 @@ func Deploy(ctx context.Context, configFilename string) error {
 	if config.Repo == "" {
 		fmt.Fprintln(os.Stderr, "No container repo specified in the config file. The container image will only be accessible locally. See `weaver kube deploy --help` for details.")
 	}
+	switch config.BuildTool {
+	case "docker", "podman":
+		break
```

Comment:
sorry for nitpicking. can you remove the `break` statement? In go each case block ends with a `break` statement by default (https://go.dev/tour/flowcontrol/9)

Reply:
No worries at all, its always good to keep syntax clean. Done.